### ISSUE TITLE
Website - About - Link to FAQ -> 404

### ISSUE THREAD

**Issue by crcdng** [hicetnunc2000/hicetnunc#435]
**Describe the bug**

The link to the FAQ on page https://www.hicetnunc.xyz/about to the FAQ is broken (404) 

Text:
"FAQ's are available here"

Link:
https://github.com/hicetnunc2000/hicetnunc/blob/main/FAQ.md

**Comment by andrevenancio** [comment]
we're aware. if you look at the closed issues you see this is just waiting on a deployment to be fixed.

**Comment by crcdng** [comment]
When I try to be helpful by reporting a currrent issue, I look for open issues. Therefore I suggest to close an issue after it is actually fixed in deployment.

**Comment by andrevenancio** [comment]
I appreciate your reporting. but I have no control over when a deployment happens. what I do have control is over triage of bugs being reported while I work on their fix. That issue has been fixed 2 days ago along with many others, and therefore I can't be checking for when the deployment happens to come here and reply to everyone and close tickets by then. I'm working pro bono on this project on my spare time. when I submit a fix I expect it to be deployed. it hasn't been deployed yet unfortunatelly, and it should have been. I appreciate your suggestion, but I dont have time to be working on new features + giving proper support to everyone submitting an issue on github. when I fix it, I trust it to be deployed and I close the issue. to actual implement your suggestion I would have to have access to deployments, or we need to have automatic deployments setup. Which I requested, but have no insight into if that's actually going to happen or no. so it both frustraste you, and me.
